08_book_titles/book.rb

```diff
@@ -0,0 +1,8 @@
+class Book
+  attr_writer :title
+
+  def title
+    lower_words = %w[or and over the a an in of]
```

Comment:
This can be moved to a constant to avoid allocating the array each time the title method is called
as:

```
class Book
  LINK_WORDS =  %w[or and over the a an in of].freeze
  # ...
end
```